### ISSUE TITLE
how to use

### ISSUE THREAD

**Issue by deRightDirection** [chrisriesgo/xamarin-forms-carouselview#42]
how can i add this control now to my project? do i have to add all the source code to my project? is it possible to wrap it up into a nuget-package?

**Comment by Rickedb** [comment]
No, there is no need to add all the source to your project, to install use: `Install-Package OpenProtocolInterpreter -Version 2.0.1`

Nuget: https://www.nuget.org/packages/openprotocolinterpreter

**Comment by Rickedb** [comment]
On package manager, type: `Install-Package Xamarin.Forms.CarouselView` to download it.
Nuget link if any doubt: https://www.nuget.org/packages/Xamarin.Forms.CarouselView/
